tests/test_text_generation_example.py

```diff
@@ -31,6 +31,7 @@
             ("meta-llama/Llama-2-7b-hf", 512, False, 8711),  # in some cases like TGI, reuse_cache isnt used
             ("stabilityai/stablelm-2-12b", 1, False, 81.842),
             ("codellama/CodeLlama-34b-hf", 1, True, 32.644),
+            ("stabilityai/stablelm-zephyr-3b", 1, False, 821.9666453354841),
```

Comment:
Let's rather use the existing baselines at line 32 and 68 (this one for Gaudi1) and update these numbers

Reply:
For stabilityai/stablelm-2-12b,

```
python run_generation.py --model_name_or_path stabilityai/stablelm-2-12b --use_kv_cache --max_new_tokens 100 --bf16 --batch_size 4 --use_hpu_graph
```

```
Input/outputs:
input 1: ('DeepSpeed is a machine learning framework',)
output 1: ('DeepSpeed is a machine learning frameworkhifts/spannableibirotting frameworks HEAP avocadoInforme Piercingumaravage Pacte-mailchimpmentsarydraughtsynchronously entertainmentucculentrumming curse mobbing Totality quotientting dress codependitures Sprightfullestablishes Pavestate agency Workers compensationnistraw.githubusercontent Upsgraded sideskickassistanceαι facilitiesetypegdIFned freelyzingguineaTOP notcherminalsity checkeredding_VERding_VERisurelynnovationals guerrementsarylandscryfall Tetraf ban',)

input 2: ('He is working on',)
output 1: ('He is working on behalfInsensitive/analyticsussionary workingmanship avaliação Cavenderwritten-downgraded DISCLAIMERNoSuchAlgorithmException molestedRevisionaryVILLE pandragon Vermi Workingmanship avaliação Cavenderwritten-downgraded DISCLAIMERNoSuchAlgorithmExceptionmeditationsally Speaking engagements Accumulatorshipsper capitaals Cavenderwritten-downgraded DISCLAIMERNoSuchAlgorithmExceptionmeditationally Speaking engagements Accumulatorshipsper capitaalsvernmentaryVILLE \u200b\u200b\u200b\u200b� inclusivenesses cavities Engagement-ringtoneenment péritochetteschooledtidals Mitchellianoth Anniversary Presentsationalists',)

input 3: ('He has a',)
output 1: ('He has a etceterpreterties Litigationally speaking engagements prospectoratechandraUIçe-mailchimpmentsarylandscryfalled etceterpreterties Litigationally Speaking engagements prospectoratechandraUIçe-mailchimpmentsaryjahrmatoriums engagements Accumulatorshipsper capitaals etceterpreterties Litigationally Speaking engagements Accumulatorialsity deeduteronomy_outline Vestiges banishedermen Artifact Flipper capitaals etceterpretertieschantmentmentaryjahrmatoriums engagements Accumulatorshipsper',)

input 4: ('He got all',)
output 1: ('He got all etceterpreter Conrad Clustered Conradionalities Clause Getty Imageshootsynchronously jadx GorHAMptonesecondensedغ VOCsid Garnering fremmelodicity ClauseoberintendentshippermissivekyrievousnessesResultadoPlotterriblemente Speaking engagements Accumulatorshipsper capitaals etceterpretertt Graduated tampshared raspidamente Speaking engagements AccumulatorshipspermissivekyrievousnessesInforme manturbationdiaries Lane avocadoInforme manturbationdiaries Lane avocadoInforme manturbationdiaries Lane',)


Stats:
---------------------------------------------------------------------------------------------------------------
Throughput (including tokenization) = 311.46584535679483 tokens/second
Number of HPU graphs                = 12
Memory allocated                    = 22.85 GB
Max memory allocated                = 22.92 GB
Total memory available              = 94.62 GB
Graph compilation duration          = 12.2312895830255 seconds
---------------------------------------------------------------------------------------------------------------
```